src/stacker/ConcreteStacker.ts

```diff
@@ -25,6 +25,27 @@ export class ConcreteStacker implements Stacker {
     this.repositoryUpdateListener(repo);
   };
 
+  private localRefPrefix : string = "refs/heads/";
+
+  private refIsLocal(refString: string) : boolean {
+    return refString.startsWith(this.localRefPrefix);
+  }
+
+  private localRefToBranchName(refString: string) : string {
+    if (!this.refIsLocal(refString)) return refString;
+    return refString.substr("refs/heads/".length);
+  }
+
+  private isSttackBranch(branch: BranchName) : boolean { 
+    if(branch.substr(0,6) === "sttack") return true; 
+    return false; 
```

Comment:
nit: you can use `startsWith`

```suggestion
    return branch.startsWith("sttack-");
```